docs: Add info to automate runtime-tests (#156)
unoplatform/uno.ui.runtimetests.engine: README.md

These attributes will ask for the runtime test engine to replace the ones defined by the `Uno.UI.RuntimeTests.Engine` package.

> ## Running the tests automatically during CI

Reviewer: We should add a note about using Uno.UITest to run the runtime tests for Android/iOS/WASM as well, even if it is hopefully temporary. We can point to current apps that do this currently

Author: Added here https://github.com/unoplatform/uno.ui.runtimetests.engine/pull/156/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R242

> When your application references the runtime-test engine, as soon as you start it with the following environement variables, the runtime-test engine will automatically run the tests on application startup and then kill the app.
> 

Reviewer 2: ```suggestion
When your application references the runtime-test engine, as soon as you start it with the following environment variables, the runtime-test engine will automatically run the tests on application startup and then kill the app.

```

* **UNO_RUNTIME_TESTS_RUN_TESTS**: This can be either
	* A json serialized [test configuration](https://github.com/unoplatform/uno.ui.runtimetests.engine/blob/main/src/Uno.UI.RuntimeTests.Engine.Library/Engine/UnitTestEngineConfig.cs) (use `{}` to run with default configuration);
 	* A filter string.
* **UNO_RUNTIME_TESTS_OUTPUT_PATH**: This is the output path of the test result file

> You can also define some other configurations variables:

Reviewer 2: ```suggestion
You can also define some other configuration variables:
```

* **UNO_RUNTIME_TESTS_OUTPUT_KIND**: Selects the kind of the test result file, possible values are `NUnit` (default) or `UnoRuntimeTests` (cf. [`TestResultKind`](https://github.com/unoplatform/uno.ui.runtimetests.engine/blob/main/src/Uno.UI.RuntimeTests.Engine.Library/Engine/ExternalRunner/RuntimeTestEmbeddedRunner.cs#L41))

> Currently the easiest way to run runtime-tests on the CI is using the skia-GTK head. Here an example of a azure pipeline configuration file:

Reviewer 2: ```suggestion
Currently, the easiest way to run runtime-tests on the CI is using the Skia-GTK head. Here is an example of an Azure pipeline configuration file:
```

    condition: always()
    inputs:
      PathtoPublish: $(build.artifactstagingdirectory)/skia-gtk-runtime-test-build.binlog
      ArtifactName: skia-runtime-test-build
      ArtifactType: Container

  - script: xvfb-run --auto-servernum --server-args='-screen 0 1280x1024x24' dotnet Uno.Extensions.RuntimeTests.Skia.Gtk.dll
    displayName: 'Run Runtime Tests (GTK)'
    workingDirectory: $(Build.SourcesDirectory)/src/Uno.Extensions.RuntimeTests/Uno.Extensions.RuntimeTests.Skia.Gtk/bin/Debug/net7.0
    env:
      UNO_RUNTIME_TESTS_RUN_TESTS: '{}'
      UNO_RUNTIME_TESTS_OUTPUT_PATH: '$(Common.TestResultsDirectory)/skia-gtk-runtime-tests-results.xml'

  - task: PublishTestResults@2
    displayName: 'Publish GTK Runtime Tests Results'
    condition: always()
    retryCountOnTaskFailure: 3
    inputs:
      testRunTitle: 'GTK Runtime Tests Run'
      testResultsFormat: 'NUnit'
      testResultsFiles: '$(Common.TestResultsDirectory)/skia-gtk-runtime-tests-results.xml'
      failTaskOnFailedTests: true 
```

Notes:
* This is running the GTK head using a virtual x-server (xvfb).
* We use `{}` for the `UNO_RUNTIME_TESTS_RUN_TESTS` in order to run all tests with default configuration.

> * If you want to test hot-relaod scenarios (usually relevant only for library developers), you need to build your test project in debug (`-c Debug`).

Reviewer 2: ```suggestion
* If you want to test hot-reload scenarios (usually relevant only for library developers), you need to build your test project in debug (`-c Debug`).
```

Author: ```suggestion
* If you want to test hot-reload scenarios (usually relevant only for library developers), you need to build your test project in debug (`-c Debug`).

### Running the tests automatically during CI on WASM and mobile targets
Alternatively, you can also run the runtime-tests on the CI using ["UI Tests"](https://github.com/unoplatform/Uno.UITest). Here is an example of how it's integrated in uno's core CI](https://github.com/unoplatform/uno/blob/master/src/SamplesApp/SamplesApp.UITests/RuntimeTests.cs#L32.
```
